### PR TITLE
added read_response_json for testing

### PR DESCRIPTION
**Pull request by Dowwie**

### CODE REVIEW

File: src/test.rs [hunk 1]
```diff
@@ -363,4 +365,55 @@ impl TestRequest {
...
+    /// 
+    ///     let payload = r#"{"id":"12345","name":"Nikolay Kim"}"#.as_bytes();
+    /// 
+    ///     let req = test::TestRequest::post()
+    ///                      .uri("/people")
+    ///                      .header(header::CONTENT_TYPE, "application/json")
+    ///                      .set_payload(payload)
+    ///                      .to_request();
+    /// 
+    ///     let result = test::read_response_json<_, _, _, Person>(&mut app, req);
```
Comment (fafhrd91) [inline src/test.rs]: change this line to:

```rust
let result: Person = test::read_response_json(&mut app, req);
```

File: src/test.rs [hunk 2]
```diff
@@ -363,4 +365,55 @@ impl TestRequest {
...
+    /// }
+    /// ```
+    pub fn read_response_json<S, R, B, T>(app: &mut S, req: R) -> T
+    where
+        B: MessageBody,
+        S: Service<Request = R, Response = ServiceResponse<B>, Error = Error>,
```
Comment (fafhrd91) [inline src/test.rs]: you can also replace `R` with `ServiceRequest`

File: src/test.rs (hunk 1 above)
Comment (Dowwie) [reply src/test.rs]: resolved

File: src/test.rs (hunk 2 above)
Comment (Dowwie) [reply src/test.rs]: done

**Review by fafhrd91 — APPROVED** [review]

**Pull request merged**